### DISCUSSION TITLE
two issue with windows 11 I can not figure out.

### DISCUSSION THREAD

**Discussion by skunx99** [Open-Shell/Open-Shell-Menu#1703] [General]
1) On a windows 11 system, when I right click on the start bar I have properties  on my other windows 11 I do not have properties. 
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/39191003/1e8dc31d-d1e1-40ba-a2f1-4920161a2931)

2) On both I do not get the quick lunch option but I see many other user made it work. That is the option to add a toolbar on the bar.

**Comment by tallman69** [comment]
> On both I do not get the quick lunch option but I see many other user made it work.


Microsoft discontinued the Quick Launch Toolbar with the Windows 11 taskbar.  You can get the Quick Launch toolbar if you install an application that activates the Windows 10 taskbar in Windows 11.  I use ExplorerPatcher  https://github.com/valinet/ExplorerPatcher to that.
